Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Stage 1 - Install dependencies and build the app
+FROM node:12.20 AS build-env
+
+RUN apt-get update 
+RUN apt-get clean
```

Comment:
I think it's bad practise to execute these in a Dockerfile.
Could you double-check?